frontend/packages/core/src/Resolver/input.tsx

```diff
@@ -41,6 +52,14 @@ const QueryResolver: React.FC<QueryResolverProps> = ({ schemas, submitHandler })
     setQueryData(convertChangeEvent(event).target.value);
   };
 
+  // If there is more than 1 schema dont enable autocomplete as that is not currently supported
+  let isAutoCompleteable = false;
+  const autoCompleteableSchemas = schemas.map(schema => schema?.metadata?.search?.autocompleteable);
+
+  if (autoCompleteableSchemas.length === 1 && autoCompleteableSchemas[0] === true) {
```

Comment:
Is it a requirement that the first schema must support autocomplete or can any of them as long as it's just one? If it can be any you can use filter above to get a list of just the schemas with autocomplete enabled and check the length of that to determine if autocomplete should be enabled

Reply:
Currently I have no way of knowing if a specific resource is enabled for autocomplete but rather is the backend configured to support auto complete eg: is the topology service enabled. Which is propagated up through the `autocomplete_enabled` field which will be present on all schemas, this acts as our feature flag. 

But yes I can use the filter as you suggest and just make sure its `>= 1`